Lib/typing.py

```diff
@@ -2929,9 +2929,9 @@ def __round__(self, ndigits: int = 0) -> T:
         pass
 
 
-def _make_nmtuple(name, fields, annotate_func, module, defaults = ()):
-    nm_tpl = collections.namedtuple(name, fields,
-                                    defaults=defaults, module=module)
+def _make_nmtuple(name, fields, annotate_func, module, defaults = (), classcell=collections._nmtuple_classcell_sentinel):
```

Comment:
line length again :-)

```suggestion
def _make_nmtuple(name, fields, annotate_func, module, defaults = (),
                  classcell=collections._nmtuple_classcell_sentinel):
```